xml/src/test/resources/configs/writebehind-cache.xml

```diff
@@ -27,9 +27,7 @@
         <ehcache:loaderwriter>
           <ehcache:class>com.pany.ehcache.integration.TestCacheLoaderWriter</ehcache:class>
         </ehcache:loaderwriter>
-        <ehcache:writebehind coalesce="false" concurrency="3" size="10">
-          <ehcache:batchsize>5</ehcache:batchsize>
-        </ehcache:writebehind>
+        <ehcache:writebehind coalesce="false" concurrency="1" size="10"/>
```

Comment:
Test makes no use of batching and this actually caused delay for no added value.